### ISSUE TITLE
Full-resolution residual_gauge: realise gauge orbits in the model's own (decoder, coordinate) parameter space — compensated symmetries as exact data-nulls, pinning from the penalty channel

### ISSUE THREAD

**Issue by SauersML** [SauersML/gam#998]
## Context (from closing #995)

The residual-gauge certificate's parameter space is the per-atom **mean decoder frame** — a deliberate, self-contained compression. #995 made its verdicts honest about that compression: the relative-curvature statistic survives a full-rank span, and the lowering-error scale calibrates the tolerance so compression artifacts are never read as pins. On a production circle fit the certificate now correctly reports "resolution-limited within this atom; refusing to pin" rather than over-claiming identification.

But "refusing to claim" is the honest *floor*, not the goal. Working #995 surfaced why the frame space can never do better: the true gauge orbits of the model class — circle phase shift, output-frame rotations, equal-ARD rotations — are **compensated** motions (latent coordinates move AND the decoder counter-rotates, e.g. `Φ(t+ε)·R(−ε)B = Φ(t)B` for the harmonic circle). Their net action on the mean frame is identically zero, so any frame-space realisation of such a generator measures compression error, not the symmetry. Proposal (i) of #995 (per-row lifts) is ill-posed for exactly these generators.

## The object to build: the certificate in the model's own parameter space

Realise the generators where the orbits actually live — the fit's free parameters `(B_k decoder coefficients, t_n latent coordinates, logits)`:

- **Compensated orbits are exact null directions of the data Jacobian by construction**: the combined (coordinate, decoder) motion changes no fitted value, so the data can never pin them — no tolerance, no calibration, exact at first order. (Caveat that must be carried per-atom: compensation requires the basis family to be closed under the group action — true for periodic harmonics under shifts and full spherical harmonics under rotations, only approximate for Duchon/TPS patches under so(d). The closure status is per-`SaeAtomBasisKind` and should be a field on the verdict, with non-closed bases degrading to the #995 calibrated path.)
- **The entire pinning content then comes from the penalty/prior channel** — the isometry penalty's curvature along the orbit (non-isometric reparameterizations cost; isometries of the pinned metric don't) and the ARD prior (unequal-axis rotations cost). This is exactly where the #981 gauge-reduction ladder says identification lives (rungs 2 and 4: the pin does the collapsing, in whichever metric it is computed), and the Theorem-2 OutputFisher arm transfers unchanged: evaluate the isometry-pin curvature root along the exact orbit in the harvested metric.
- **The data channel remains load-bearing only for non-class-symmetry directions** — the continuous atom-mixing interpolations behind the `Sym(F)` permutation factor — which is precisely the one family the #981 exchangeable-pair tests exercise against data today.

## Why not just keep the calibrated frame certificate

It cannot distinguish "this atom's within-gauge is genuinely pinned by the isometry penalty" from "this atom's lowering is too lossy to tell" whenever the decoder is curved — and curved decoders are the entire point of manifold atoms. The production circle, the flagship case, currently gets the honest shrug. The parameter-space certificate gives it a real verdict: phase shift unpinned without an isometry pin, pinned-up-to-O(2) with one, pinned-to-trivial under the functional-metric pin (#980 rung 4) — each as an exact statement, not a tolerance call.

## Acceptance

1. Production single-circle fit (the `tests/sae_replicate_gauge_agreement.rs` DGP): the parameter-space certificate must report the U(1) phase generator as an **exact** data-null (fraction at the noise floor, no lowering-error calibration involved), unpinned without the isometry pin, and the report's group must equal the O(2) factor the replicates demonstrably differ by — closing the loop the frame certificate can only refuse.
2. The #995 falsifier inverts: with exact-orbit realisation the mixed-curvature verdict must come from the *penalty* root alone; deleting the penalty rows must flip it unpinned with the data rows present.
3. Basis-closure honesty: a Duchon patch atom must carry the non-closed-basis flag and degrade to the calibrated #995 path, never silently pretend exactness.
4. Frame certificate stays as the cheap fallback; the parameter-space path activates when the term can supply `(B, t)` views (magic-by-default, no flags).

Cross-refs: #995 (the calibrated floor this builds past), #981 (the ladder whose rungs this realises exactly), #980 (the functional-metric pin evaluated along exact orbits), #972 (frame-factor canonicalization interacts with the decoder-coefficient realisation).

**Comment by SauersML** [comment]
Closing — landed (b36479704 + fc050116d), with one improvement over the issue's own acceptance item 3, explained below.

**The realisation.** `AtomParameterView` carries what the fit actually optimizes — basis values Φ, first-derivative jet Φ′, decoder B, latent coordinates t, assignment mass — and `residual_gauge_exact()` realises each within-atom orbit exactly: the coordinate motion field `δt` comes from the group action (circle/torus shifts chart-free, flat-patch so(d) rotations `δt_n = A t_n` — the charts the group acts on **linearly**, so the field is exact rather than a linearisation), and the decoder compensation `δB` is **profiled out by least squares** against the per-row data motion `u_n = a_n (Φ′_n B) δt_n` through one shared SVD pseudo-inverse of the activation-weighted basis. The compensation residual `r²/‖u‖²` is the orbit's true relative data cost: exactly zero for a basis closed under the action (`Φ(t+ε) = Φ(t)R(ε)` for harmonics), genuinely positive otherwise. Compensated symmetries are data-nulls **by construction** — no tolerance, no calibration (`lowering_error_scale = 0` on every exact verdict).

**The pinning channel.** All pinning of true model-class symmetries flows through the injectable per-atom `OrbitPenaltyOperator` (`apply(δB, δt) → penalty-root image`, plus its `σ_max²` stiffness), reported on the same relative-curvature convention — exactly where the #981 ladder says identification lives (rungs 2/4: the pin does the collapsing, in whichever metric it is computed). Per the amended #980 dispatch rule the data channel is the activation-reconstruction objective in its own inner product; output-Fisher reaches gauge verdicts only through this operator.

**Merging.** Viewed atoms' within-atom families are certified exactly and their lossy frame-space lifts are *replaced* (never double-reported); cross-atom families (output rotations, permutations) and unviewed atoms (spheres — nonlinear chart action — and structurally inconsistent atoms) keep the #995 calibrated frame path. `fit_diagnostics_report` routes through the exact path magic-by-default whenever no isometry pin is active; the pinned production case stays on the calibrated floor until the pin's orbit-space curvature operator is lowered (it needs the second-derivative jets) — documented at the call site, not silent.

**Acceptance:**
1. *Production circle* — the `tests/sae_replicate_gauge_agreement.rs` oracle now **positively certifies** what the frame certificate could only refuse: the U(1) phase generator is unpinned with its compensation residual at the noise floor (≤ 1e−6), zero calibration, nonzero orbit norm, and `Isom(M_k)` named in the certified group — the exact freedom the replicates demonstrably differ by in the alignment block of the same test.
2. *Falsifier inversion* — `tests/sae_exact_orbit_certificate.rs::penalty_channel_alone_decides_true_symmetries`: the phase orbit is a data-null either way; a drift-costing pin operator alone pins it (fraction exactly 1) and removing the operator flips it unpinned with the data untouched.
3. *Basis-closure honesty* — implemented **stronger than specced**: instead of a declared per-`SaeAtomBasisKind` flag with degradation, closure is *computed* — the LS residual is the per-generator number. `tests/sae_exact_orbit_certificate.rs::basis_closure_is_computed_not_declared`: a linear patch basis is exactly null under so(2); a quadratic basis (the Duchon-like case — rotating the chart produces a `t₁t₂` term outside the span, analytic residual fraction ≈ 0.48 on the planted grid) is **genuinely data-pinned**. That verdict is not a degradation to apologize for: a basis not closed under the action means the rotation is *not a symmetry of the model class*, and pinning it is the truth. Nothing pretends exactness anywhere: every exact verdict carries its computed residual.
4. *Fallback* — the frame certificate remains the cheap path and the only path for spheres and pin-active fits; activation is derived from the fit, no flags.

The ill-posedness that motivated this issue — frame-space lifts of compensated orbits measuring compression instead of symmetry — is now structurally gone for every linearly-acting chart: those generators never touch frame space at all.

**Comment by SauersML** [comment]
**Follow-up: the remaining pin-ACTIVE rung is landed (487a710d0) — the exact path now covers pinned fits too.**

The close-out left one seam: `fit_diagnostics_report` routed the **pin-active** case to the calibrated frame path (the honest floor), because the exact orbit path needed the isometry pin's *orbit-space curvature operator*, "which needs the second-derivative jets". That operator is now lowered.

**The operator.** `isometry_orbit_penalty_operator(view, μ)` builds the `OrbitPenaltyOperator` from the atom's second jet Φ''. The isometry penalty is `P = ½μ Σ_n ‖g_n − g_ref‖²_F` on the pullback first-fundamental gram `g_n = J_nᵀ J_n`, `J_n = Φ'_n B`. At a converged isometric fit the residual `g_n−g_ref ≈ 0`, so the penalty's curvature along an orbit `(δB, δt)` is the Gauss-Newton term `μ Σ_n ‖δg_n‖²_F`, and the curvature-root image is `√μ·{δg_n}`. The first-order gram change

```
δJ_n[i,c] = Σ_m Φ'_n[m,c] δB[m,i]  +  Σ_{m,e} Φ''_n[m,c,e] δt_n[e] B[m,i]
δg_n[a,b] = Σ_i ( δJ_n[i,a] J_n[i,b] + J_n[i,a] δJ_n[i,b] )
```

**differentiates J through t via the second jet Φ''** — exactly the second jets the seam named, and exactly what the frame path (no Φ'') could not supply. The reported relative-curvature fraction `cost/stiffness²` is invariant to the pin strength μ (both faces scale with μ), so the operator is built at canonical unit weight.

**Routing.** `AtomParameterView` now carries `basis_second_jet` (sourced from the evaluator's analytic Hessian); `fit_diagnostics_report` builds a per-atom isometry operator for every viewed atom when the pin is active and routes through `residual_gauge_exact` (not the frame floor). An atom whose basis exposes no analytic Hessian supplies no operator and falls back to the data residual — never an error, magic-by-default.

**Conservatism (the verdict the seam worried about — "over-claiming freedom").** `tests/sae_exact_orbit_certificate.rs::pin_active_orbit_operator_from_second_jet_is_conservative`: a genuine model-class isometry (the harmonic circle's phase shift; a flat-patch so(2) rotation) gets **zero** orbit cost (δg=0) and stays a certified freedom even under the pin — the operator never spuriously pins; a non-isometric orbit (a quadratic patch not closed under so(2)) stays pinned — the pin-active exact path never spuriously frees it. The certificate can only ever *cost* an orbit, never wrongly free one.

`src/sae_identifiability.rs`, `src/terms/sae_manifold.rs`. The ill-posedness is now structurally gone for pinned fits too: within-atom families are certified on their exact orbits whether or not an isometry pin is active.
